### PR TITLE
Added more information on Okta hosted and Customer hosted concepts

### PR DESCRIPTION
**Pull request by christinechoy-okta**
## Description:
- **What's changed?** Added more information on Okta hosted and customer hosted
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-320492](https://oktainc.atlassian.net/browse/OKTA-320492)

### CODE REVIEW

**Comment by christinechoy-okta** [comment]
Aaron Parecki is the technical reviewer but doesn't have a Github account. He's reviewed this via Slack.

File: packages/@okta/vuepress-site/docs/concepts/hosted-vs-embedded/index.md [hunk 1]
```diff
@@ -4,7 +4,20 @@ title: Okta Hosted Vs. Customer Hosted
 
 ## Okta Hosted Vs. Customer Hosted
 
-Okta offers multiple integration options when designing an authentication experience. The two most recommended concepts are using the Okta-hosted Sign-In Widget and the Customer-hosted Sign-In Widget. Both of these Okta-built experiences provide easy implementation and maintenance, industry best practice, user self service, and built-in user registration. The difference between the two concepts is where the Sign-In Widget itself is hosted.
+Okta offers multiple integration options when designing an authentication experience. The two most recommended concepts are using the Okta-hosted Sign-In Widget and the Customer-hosted Sign-In Widget. Both of these Okta-built experiences provide easy implementation and maintenance, industry best practice, user self service, and built-in user registration. You can customize the Okta URL domain and the Okta-hosted or Customer-hosted Sign-In Widget style. See [Customize the Okta URL domain](/docs/guides/custom-url-domain/overview/) and [Style the Widget](/docs/guides/style-the-widget/before-you-begin/).
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/concepts/hosted-vs-embedded/index.md:7]:
```suggestion
Okta offers multiple integration options for designing an authentication experience. The two most recommended options are integrating the Okta-hosted Sign-In Widget and the customer-hosted Sign-In Widget. Both of these Okta-built experiences provide easy implementation and maintenance, industry best practice, user self service, and built-in user registration. You can customize the Okta URL domain and the Okta-hosted or customer-hosted Sign-In Widget style. See [Customize the Okta URL domain](/docs/guides/custom-url-domain/overview/) and [Style the Widget](/docs/guides/style-the-widget/before-you-begin/).
```

Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/concepts/hosted-vs-embedded/index.md:7]: Not sure if "Customer-hosted Sign-In Widget" is a product? I am assuming that "Sign-In Widget" is the product and that customer-hosted is the modifier. Change this if my assumptions are incorrect. Thanks!

File: packages/@okta/vuepress-site/docs/concepts/hosted-vs-embedded/index.md [hunk 2]
```diff
@@ -4,7 +4,20 @@ title: Okta Hosted Vs. Customer Hosted
...
+
+The difference between the two concepts is where the Sign-In Widget itself is hosted. See [Okta-hosted widget](/docs/concepts/hosted-vs-embedded/#okta-hosted-widget) and [Customer-hosted widget](/docs/concepts/hosted-vs-embedded/#customer-hosted-widget) for diagrams on where the Sign-In Widget is hosted.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/concepts/hosted-vs-embedded/index.md:9]:
```suggestion
The difference between the two integration options is where the Sign-In Widget is hosted. See [Okta-hosted widget](#okta-hosted-widget) and [Customer-hosted widget](#customer-hosted-widget) for hosted scenarios.
```

Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/concepts/hosted-vs-embedded/index.md:9]: I didn't see diagrams in the topic. If the diagrams are to be added, then please revert this suggestion. Thanks

File: packages/@okta/vuepress-site/docs/concepts/hosted-vs-embedded/index.md [hunk 3]
```diff
@@ -4,7 +4,20 @@ title: Okta Hosted Vs. Customer Hosted
...
+
+Another difference is when you're using the Okta-hosted Sign-In widget, you're communicating with Okta using OAuth and OIDC. When you're using the Customer-hosted Sign-In widget, you're using the Javascript SDK and communication with Okta is handled for you transparently.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/concepts/hosted-vs-embedded/index.md:11]:
```suggestion
Another difference between the two options is the method of communication. When you're using the Okta-hosted Sign-In Widget, you're communicating with Okta using OAuth 2.0 and OIDC. When you're using the customer-hosted Sign-In Widget, you're using the Okta JavaScript SDK and communication with Okta is handled for you transparently.
```

File: packages/@okta/vuepress-site/docs/concepts/hosted-vs-embedded/index.md [hunk 4]
```diff
@@ -4,7 +4,20 @@ title: Okta Hosted Vs. Customer Hosted
...
+
+## Configure policies for app sign-in experiences
+
+For both Okta-hosted and Customer-hosted sign-in experiences, you can configure the policies in the Okta Admin console to determine how the app will behave.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/concepts/hosted-vs-embedded/index.md:15]:
```suggestion
For both Okta-hosted and customer-hosted sign-in experiences, you can configure policies in the Okta Admin Console to determine the behavior of the app. 
```

File: packages/@okta/vuepress-site/docs/concepts/hosted-vs-embedded/index.md [hunk 5]
```diff
@@ -4,7 +4,20 @@ title: Okta Hosted Vs. Customer Hosted
...
+
+Another difference is when you're using the Okta-hosted Sign-In widget, you're communicating with Okta using OAuth and OIDC. When you're using the Customer-hosted Sign-In widget, you're using the Javascript SDK and communication with Okta is handled for you transparently.
+
+## Configure policies for app sign-in experiences
+
+For both Okta-hosted and Customer-hosted sign-in experiences, you can configure the policies in the Okta Admin console to determine how the app will behave. 
+
+1. Go to **Security** > **API**. 
+1. In the **Authorization Servers** tab, select an authorization server and click the edit icon beside it. 
+1. On the authorization server page, select the **Access Policies** tab and scroll down to the Rules
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/concepts/hosted-vs-embedded/index.md:19]:
```suggestion
1. On the authorization server page, select the **Access Policies** tab and scroll down to the Rules list.
```

File: packages/@okta/vuepress-site/docs/concepts/hosted-vs-embedded/index.md [hunk 6]
```diff
@@ -4,7 +4,20 @@ title: Okta Hosted Vs. Customer Hosted
...
+1. Select a Rule, and click the edit icon beside it. The Edit Rule window appears where you can update the settings for the access policies.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/concepts/hosted-vs-embedded/index.md:20]:
```suggestion
1. Click the edit icon beside the Rule you want to modify. The Edit Rule window appears where you can update the settings for the access policies.
```

**Review by vanngo-okta — CHANGES_REQUESTED** [review]
Nice addition to this topic! I added some suggestions, but please ignore if the assumptions are not correct. Thanks.

File: packages/@okta/vuepress-site/docs/concepts/hosted-vs-embedded/index.md (hunk 2 above)
Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/concepts/hosted-vs-embedded/index.md:9]: The diagrams are in the sections as referenced.

**Review by vanngo-okta — APPROVED** [review]
LGTM 🚢

**Comment by aaronpk** [comment]
👋 I do have a github account!

**Pull request merged**